assets/src/containers/AssignmentPlanning.js

```diff
@@ -39,7 +39,7 @@ const styles = theme => ({
 
 const getCurrentWeek = assignmentData => {
   // default week
-  let currentWeek = 0
+  let currentWeek = null
```

Comment:
if 0 is returned there is a case that if assignments don't have a due date and no current week the table will have `week 0` row. In that case `week 0` is color coded, which we don't want as it doesn't make sense.

Reply:
Got it, thanks, this makes more sense.